libs/audit-service/src/main/java/uk/gov/di/ipv/core/library/auditing/extension/AuditExtensionAccountIntervention.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.di.ipv.core.library.auditing.extension;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import uk.gov.di.ipv.core.library.annotations.ExcludeFromGeneratedCoverageReport;
+
+@ExcludeFromGeneratedCoverageReport
+@Getter
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class AuditExtensionAccountIntervention implements AuditExtensions {
+
+    private static final String REPROVE_IDENTITY_TYPE = "reprove_identity";
+
+    @JsonProperty private Boolean success;
+    @JsonProperty private final String type;
+
+    private AuditExtensionAccountIntervention(String type) {
+        this.type = type;
+    }
+
+    private AuditExtensionAccountIntervention(String type, Boolean success) {
+        this.type = type;
+        this.success = success;
+    }
+
+    public static AuditExtensionAccountIntervention
+            newReproveIdentityAuditExtensionAccountIntervention() {
```

Comment:
You probably don't need the duplicate `AuditExtensionAccountIntervention` since the class qualifies it already. E.g.:
```
AuditExtensionAccountIntervention.reproveIdentity();
```
(Feel free to add a verb if you like)